### ISSUE TITLE
Light Theme: Embedded media in main body disappearing

### ISSUE THREAD

**Issue by meetinghouse** [meetinghouse/cms#190]
I've put a bit of HTML in the main body of a portfolio to make a Pay Now button linked to the client's Paypal. It seems to work fine, and all is well when I save it and test it.

But if I go back to the edit screen for that portfolio page the embedded button no longer appears. So if I save the page,  the button is totally gone. The embed code isn't visible in the Source popup, either, just a small portion of the image HTML, not the whole thing. I'm not sure what's happening, but here are the details:

Here's how it's supposed to look:
http://historicstructuresdc.corbettresearchgroup.com/payment
![screenshot 2015-08-25 at 9 35 48 am](https://cloud.githubusercontent.com/assets/5215893/9467905/bab1b3ba-4b0c-11e5-9508-47d7333b0a5a.png)

If you go to the above URL, you'll see that the button is no longer there, now that I've hit Update on that page's edit screen.

This website last had code re-deployed to it on 8-18, in case that is relevant. It's pretty up-to-date.

Here is the embed code I've been using, in case that is the source of the problem.

<form action="https://www.paypal.com/cgi-bin/webscr" method="post">
<input type="hidden" name="cmd" value="_s-xclick">
<input type="image" src="http://www.historicstructuresdc.com/paynow.jpg" border="0" name="submit" alt="Make payments with PayPal - it's fast, free and secure!">
<img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
<input type="hidden" name="encrypted" value="-----BEGIN PKCS7-----MIIHRwYJKoZIhvcNAQcEoIIHODCCBzQCAQExggEwMIIBLAIBADCBlDCBjjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MRQwEgYDVQQKEwtQYXlQYWwgSW5jLjETMBEGA1UECxQKbGl2ZV9jZXJ0czERMA8GA1UEAxQIbGl2ZV9hcGkxHDAaBgkqhkiG9w0BCQEWDXJlQHBheXBhbC5jb20CAQAwDQYJKoZIhvcNAQEBBQAEgYAyuigsiIRmpxF9Lv8w1QqNoZ70m1p0KGtxvjqEA26mjY0oz/Yc1s7B/jXXVmJElmGJtETXAjdQDMypZ9FoFhfEwvqhhbSEBUexaZJJU56sCXdTmIa8PaM1he1az9Zk0hSz1mKcIWeR0RlsrKBa9oNwybEuLw5Wpn8n31wg+j+iYzELMAkGBSsOAwIaBQAwgcQGCSqGSIb3DQEHATAUBggqhkiG9w0DBwQIf+AjDs4fx3WAgaAl4tYNiy7QWyI4cUbzg/BhvhiYrLycPfRTIiBY4TGrE1tu4tjUSbOwn0xN2ow46KAS9W53x83Q7hpAThLBxssfLYHtOokTS01ZJnIBR3IYLX+EtFKjfAJNYHZG9nAxGbqnJ7nJUIZoQIBWuo39Z3HxG/2Paf9z7DSE2qDFkfc7JPz5+DfaC4Rcwt3nEiOGnFStswbI5gkZ17U0EUd1DnpGoIIDhzCCA4MwggLsoAMCAQICAQAwDQYJKoZIhvcNAQEFBQAwgY4xCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEUMBIGA1UEChMLUGF5UGFsIEluYy4xEzARBgNVBAsUCmxpdmVfY2VydHMxETAPBgNVBAMUCGxpdmVfYXBpMRwwGgYJKoZIhvcNAQkBFg1yZUBwYXlwYWwuY29tMB4XDTA0MDIxMzEwMTMxNVoXDTM1MDIxMzEwMTMxNVowgY4xCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEUMBIGA1UEChMLUGF5UGFsIEluYy4xEzARBgNVBAsUCmxpdmVfY2VydHMxETAPBgNVBAMUCGxpdmVfYXBpMRwwGgYJKoZIhvcNAQkBFg1yZUBwYXlwYWwuY29tMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDBR07d/ETMS1ycjtkpkvjXZe9k+6CieLuLsPumsJ7QC1odNz3sJiCbs2wC0nLE0uLGaEtXynIgRqIddYCHx88pb5HTXv4SZeuv0Rqq4+axW9PLAAATU8w04qqjaSXgbGLP3NmohqM6bV9kZZwZLR/klDaQGo1u9uDb9lr4Yn+rBQIDAQABo4HuMIHrMB0GA1UdDgQWBBSWn3y7xm8XvVk/UtcKG+wQ1mSUazCBuwYDVR0jBIGzMIGwgBSWn3y7xm8XvVk/UtcKG+wQ1mSUa6GBlKSBkTCBjjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MRQwEgYDVQQKEwtQYXlQYWwgSW5jLjETMBEGA1UECxQKbGl2ZV9jZXJ0czERMA8GA1UEAxQIbGl2ZV9hcGkxHDAaBgkqhkiG9w0BCQEWDXJlQHBheXBhbC5jb22CAQAwDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQUFAAOBgQCBXzpWmoBa5e9fo6ujionW1hUhPkOBakTr3YCDjbYfvJEiv/2P+IobhOGJr85+XHhN0v4gUkEDI8r2/rNk1m0GA8HKddvTjyGw/XqXa+LSTlDYkqI8OwR8GEYj4efEtcRpRYBxV8KxAW93YDWzFGvruKnnLbDAF6VR5w/cCMn5hzGCAZowggGWAgEBMIGUMIGOMQswCQYDVQQGEwJVUzELMAkGA1UECBMCQ0ExFjAUBgNVBAcTDU1vdW50YWluIFZpZXcxFDASBgNVBAoTC1BheVBhbCBJbmMuMRMwEQYDVQQLFApsaXZlX2NlcnRzMREwDwYDVQQDFAhsaXZlX2FwaTEcMBoGCSqGSIb3DQEJARYNcmVAcGF5cGFsLmNvbQIBADAJBgUrDgMCGgUAoF0wGAYJKoZIhvcNAQkDMQsGCSqGSIb3DQEHATAcBgkqhkiG9w0BCQUxDxcNMDcwODA3MjAwNTE4WjAjBgkqhkiG9w0BCQQxFgQU0yAoCE2M8RJTHBLke3gT0xM2JdgwDQYJKoZIhvcNAQEBBQAEgYCm2J4QtfBbRjk3yWJiQ1tmL6MBhhaPEh7/mugP1oejF2VlSRx5A6Z2LBvS+XW7/0GYiQ9qXkYV7m4Iyl0LvgDsoydh6o+ohldJc/bIkhm89eFsWQXyKfaqbdOLGY5QYoTbEurMXcEX2Fzjz2EFkdCisZRjT/yGxj7cslzGYyzY+w==-----END PKCS7-----
">
</form>

**Comment by acavanagh** [comment]
Looks like github stripped out the code as well.  I'm guessing there's some
javascript in there, which wysiwyg editors strip out.  Try pasting the code
into github between sets of three backticks, like this:

```
CODE HERE
```

**Comment by meetinghouse** [comment]
```
<form action="https://www.paypal.com/cgi-bin/webscr" method="post">
<input type="hidden" name="cmd" value="_s-xclick">
<input type="image" src="http://www.historicstructuresdc.com/paynow.jpg" border="0" name="submit" alt="Make payments with PayPal - it's fast, free and secure!">
<img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
<input type="hidden" name="encrypted" value="-----BEGIN PKCS7-----MIIHRwYJKoZIhvcNAQcEoIIHODCCBzQCAQExggEwMIIBLAIBADCBlDCBjjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MRQwEgYDVQQKEwtQYXlQYWwgSW5jLjETMBEGA1UECxQKbGl2ZV9jZXJ0czERMA8GA1UEAxQIbGl2ZV9hcGkxHDAaBgkqhkiG9w0BCQEWDXJlQHBheXBhbC5jb20CAQAwDQYJKoZIhvcNAQEBBQAEgYAyuigsiIRmpxF9Lv8w1QqNoZ70m1p0KGtxvjqEA26mjY0oz/Yc1s7B/jXXVmJElmGJtETXAjdQDMypZ9FoFhfEwvqhhbSEBUexaZJJU56sCXdTmIa8PaM1he1az9Zk0hSz1mKcIWeR0RlsrKBa9oNwybEuLw5Wpn8n31wg+j+iYzELMAkGBSsOAwIaBQAwgcQGCSqGSIb3DQEHATAUBggqhkiG9w0DBwQIf+AjDs4fx3WAgaAl4tYNiy7QWyI4cUbzg/BhvhiYrLycPfRTIiBY4TGrE1tu4tjUSbOwn0xN2ow46KAS9W53x83Q7hpAThLBxssfLYHtOokTS01ZJnIBR3IYLX+EtFKjfAJNYHZG9nAxGbqnJ7nJUIZoQIBWuo39Z3HxG/2Paf9z7DSE2qDFkfc7JPz5+DfaC4Rcwt3nEiOGnFStswbI5gkZ17U0EUd1DnpGoIIDhzCCA4MwggLsoAMCAQICAQAwDQYJKoZIhvcNAQEFBQAwgY4xCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEUMBIGA1UEChMLUGF5UGFsIEluYy4xEzARBgNVBAsUCmxpdmVfY2VydHMxETAPBgNVBAMUCGxpdmVfYXBpMRwwGgYJKoZIhvcNAQkBFg1yZUBwYXlwYWwuY29tMB4XDTA0MDIxMzEwMTMxNVoXDTM1MDIxMzEwMTMxNVowgY4xCzAJBgNVBAYTAlVTMQswCQYDVQQIEwJDQTEWMBQGA1UEBxMNTW91bnRhaW4gVmlldzEUMBIGA1UEChMLUGF5UGFsIEluYy4xEzARBgNVBAsUCmxpdmVfY2VydHMxETAPBgNVBAMUCGxpdmVfYXBpMRwwGgYJKoZIhvcNAQkBFg1yZUBwYXlwYWwuY29tMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDBR07d/ETMS1ycjtkpkvjXZe9k+6CieLuLsPumsJ7QC1odNz3sJiCbs2wC0nLE0uLGaEtXynIgRqIddYCHx88pb5HTXv4SZeuv0Rqq4+axW9PLAAATU8w04qqjaSXgbGLP3NmohqM6bV9kZZwZLR/klDaQGo1u9uDb9lr4Yn+rBQIDAQABo4HuMIHrMB0GA1UdDgQWBBSWn3y7xm8XvVk/UtcKG+wQ1mSUazCBuwYDVR0jBIGzMIGwgBSWn3y7xm8XvVk/UtcKG+wQ1mSUa6GBlKSBkTCBjjELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAkNBMRYwFAYDVQQHEw1Nb3VudGFpbiBWaWV3MRQwEgYDVQQKEwtQYXlQYWwgSW5jLjETMBEGA1UECxQKbGl2ZV9jZXJ0czERMA8GA1UEAxQIbGl2ZV9hcGkxHDAaBgkqhkiG9w0BCQEWDXJlQHBheXBhbC5jb22CAQAwDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQUFAAOBgQCBXzpWmoBa5e9fo6ujionW1hUhPkOBakTr3YCDjbYfvJEiv/2P+IobhOGJr85+XHhN0v4gUkEDI8r2/rNk1m0GA8HKddvTjyGw/XqXa+LSTlDYkqI8OwR8GEYj4efEtcRpRYBxV8KxAW93YDWzFGvruKnnLbDAF6VR5w/cCMn5hzGCAZowggGWAgEBMIGUMIGOMQswCQYDVQQGEwJVUzELMAkGA1UECBMCQ0ExFjAUBgNVBAcTDU1vdW50YWluIFZpZXcxFDASBgNVBAoTC1BheVBhbCBJbmMuMRMwEQYDVQQLFApsaXZlX2NlcnRzMREwDwYDVQQDFAhsaXZlX2FwaTEcMBoGCSqGSIb3DQEJARYNcmVAcGF5cGFsLmNvbQIBADAJBgUrDgMCGgUAoF0wGAYJKoZIhvcNAQkDMQsGCSqGSIb3DQEHATAcBgkqhkiG9w0BCQUxDxcNMDcwODA3MjAwNTE4WjAjBgkqhkiG9w0BCQQxFgQU0yAoCE2M8RJTHBLke3gT0xM2JdgwDQYJKoZIhvcNAQEBBQAEgYCm2J4QtfBbRjk3yWJiQ1tmL6MBhhaPEh7/mugP1oejF2VlSRx5A6Z2LBvS+XW7/0GYiQ9qXkYV7m4Iyl0LvgDsoydh6o+ohldJc/bIkhm89eFsWQXyKfaqbdOLGY5QYoTbEurMXcEX2Fzjz2EFkdCisZRjT/yGxj7cslzGYyzY+w==-----END PKCS7-----
">
</form>
```

**Comment by acavanagh** [comment]
This should be all set now.  The text editor is set to strip out tags that might be suspicious.  I told it that form and input tags aren't suspicious.  Since only authenticated users have access this is pretty safe.  Button works now, and will for any updated site.

**Comment by meetinghouse** [comment]
Hmm, I've tested this (by going to that same page and opening the edit page, then re-saving it) and it disappeared again.

**Comment by acavanagh** [comment]
Try clearing your browser cache before you edit it - the setting that I changed is javascript, which can be cached by the browser.  I just added the code, saved the page, confirmed that it was visible on the page, edited the page again, saved again, and it was still visible.  I wasn't able to do that before I changed the javascript setting, so I'm pretty sure that it's working now because of that change.  If you clear your cache and the problem persists for you then I'll look into possible issues with operating system/browser differences.
